src/rf/rf/settings/base.py

```diff
@@ -59,20 +59,10 @@ def get_env_setting(setting):
 # See: https://docs.djangoproject.com/en/dev/ref/settings/#caches
 CACHES = {
     'default': {
-        # The Redis database at index 0 is used by Logstash/Beaver
-        'BACKEND': 'django_redis.cache.RedisCache',
-        'LOCATION': 'redis://{0}:{1}/1'.format(
-            environ.get('RF_CACHE_HOST', 'localhost'),
```

Comment:
These environment variable files that are consumed by `envdir` are still being written to disk.